collections/search/index.php

```diff
@@ -241,139 +243,139 @@
 										</label>
 
 										<label for="upperlat_NS" class="input-text--outlined">
-											<span class="skip-link">Select upper lat direction N/S</span>
+											<span class="skip-link"><?php echo $LANG['SELECT_UPPER_LAT_DIRECTION'] ?></span>
 											<select class="mt-1" id="upperlat_NS" name="upperlat_NS">
-												<option value="">Select N/S</option>
-												<option id="ulN" value="N">N</option>
-												<option id="ulS" value="S">S</option>
+												<option value=""><?php echo $LANG['SELECT_N_S'] ?></option>
+												<option id="ulN" value="N"><?php echo $LANG['N'] ?></option>
+												<option id="ulS" value="S"><?php echo $LANG['S'] ?></option>
 											</select>
 										</label>
 								</div>
 								<div class="input-text-container">
 									<label for="bottomlat" class="input-text--outlined">
-										<span class="skip-link">Bottom latitude</span>
-										<input type="number" step="any" min="-90" max="90" id="bottomlat" name="bottomlat" data-chip="Bottom Lat">
-										<span data-label="_Southern Latitude"></span>
-										<span class="assistive-text">Values between -90 and 90.</span>
+										<span class="skip-link"><?php echo $LANG['BOTTOM_LATITUDE'] ?></span>
+										<input type="number" step="any" min="-90" max="90" id="bottomlat" name="bottomlat" data-chip="<?php echo $LANG['BOTTOM_LAT'] ?>">
+										<span data-label="<?php echo $LANG['_SOUTHERN_LATITUDE'] ?>"></span>
+										<span class="assistive-text"><?php echo $LANG['VALUE_BETWEEN_NUM'] ?></span>
 									</label>
 									<label for="bottomlat_NS">
-										<span class="skip-link">Select bottom lat direction N/S</span>
+										<span class="skip-link"><?php echo $LANG['SELECT_BOTTOM_LAT_DIREC'] ?></span>
 										<select class="mt-1" id="bottomlat_NS" name="bottomlat_NS">
-											<option value="">Select N/S</option>
-											<option id="blN" value="N">N</option>
-											<option id="blS" value="S">S</option>
+											<option value=""><?php echo $LANG['SELECT_N_S'] ?></option>
+											<option id="blN" value="N"><?php echo $LANG['N'] ?></option>
+											<option id="blS" value="S"><?php echo $LANG['S'] ?></option>
 										</select>
 									</label>
 								</div>
 								<div class="input-text-container">
 									<label for="leftlong" class="input-text--outlined">
-										<span class="skip-link">Left longitude</span>
-										<input type="number" step="any" min="-180" max="180" id="leftlong" name="leftlong" data-chip="Left Long">
-										<span data-label="_Western Longitude"></span>
-										<span class="assistive-text">Values between -180 and 180.</span>
+										<span class="skip-link"><?php echo $LANG['LEFT_LONGTITUDE'] ?></span>
+										<input type="number" step="any" min="-180" max="180" id="leftlong" name="leftlong" data-chip="<?php echo $LANG['LEFT_LONG'] ?>">
+										<span data-label="<?php echo $LANG['_WESTERN_LONTITUDE'] ?>"></span>
```

Comment:
I think this underscore at the beginning is an accident. Can you remove from the lang tag (and fix in lang file)? Also, "longitude" is misspelled.

Reply:
@Atticus29 , can you advise whether underscores at the beginning like this are intentional? I'm unfamiliar with the "data-label" tag.

Reply:
Intentional. I tried again to get the styling to work without them. Because the label is being added in a `content:` attribute, adjusting the styling has proven to be really frustrating. I've spent too much time trying to fix this twice now haha. I'm content just using the underscores to fix the issue. The other option would be re-tooling some of the styling that NEON has implemented on this. This seems like overkilll for an issue that an underscore is solving.